src/Microsoft.TemplateEngine.Orchestrator.RunnableProjects/ConfigModel/BaseValueSymbol.cs

```diff
@@ -69,7 +69,7 @@ private protected BaseValueSymbol(string name, string? replaces) : base(name, re
         /// </summary>
         public string? DataType { get; internal init; }
 
-        protected bool TryGetIsRequiredField(JToken token, out bool result)
+        private protected bool TryGetIsRequiredField(JToken token, out bool result)
```

Comment:
Oh I wasn't aware this is possible - apparently it is :-) https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/keywords/private-protected